qiskit-superstaq/examples/resource_estimate.ipynb

```diff
@@ -113,42 +113,42 @@
   },
```

Comment:
<p>Could just be the notebook viewer formatting, but is there a lot of whitespace in the output before the circuit print? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/728/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Edit: Appears only in NBviewer</p>